setup.py

```diff
@@ -56,6 +56,12 @@
           'schema-salad >= 2.6.20170927145003, < 3',
           'typing >= 3.5.3',
           'six >= 1.8.0',
+          'prov == 1.5.1',
+          'pathlib == 1.0.1',
```

Comment:
pathlib is deprecated, probably should use pathlib2

https://pypi.python.org/pypi/pathlib/
https://pypi.python.org/pypi/pathlib2/

Reply:
resolved